index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <script type="module" src="scripts/index.js"></script>
+    <title>Haker_news_JS</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="container__header">
+        <div class="logo">
+          <div class="logo__image"><img src="images/logo.gif" /></div>
+          <h1 class="logo__title">Hacker News</h1>
+        </div>
+        <nav>
+          <ul class="menu">
+            <li class="menu__item">
+              <a id="user" href="" class="">User info</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <div class="container__body">
+        <div>
```

Comment:
Lets try to clean a little bit html and remove redundant divs 
`<ol id="news-list" type="1" class="generic-list">
          </ol>
          <form class="form" name="user">`